### DISCUSSION TITLE
Codespace not available

### DISCUSSION THREAD

**Discussion by Nightmare-crypto** [community/community#86656] [Programming Help]
### Select Topic Area

Question

### Body

Im trying to use NOVNC's codespace but it just say's "Codespace Not available."
Please Help me!

**Comment by Nightmare-crypto** [comment]
![Image 1-11-24 at 3 42 PM](https://github.com/community/community/assets/113560903/796a7bdf-66a9-4b2a-9d94-68d3447735a0)
Here is a image

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Nightmare-crypto ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by Nightmare-crypto** [reply]
Umm still not working after a day

**Comment by Nightmare-crypto** [comment]
> Thanks for posting in the GitHub Community, @Nightmare-crypto ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.
> 
> If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:
> 
> * Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
> * Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
> * Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.
> 
> All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by Nightmare-crypto** [comment]
......

**Comment by mishmanners** [comment]
You'll need to check:
- do you have access to the Codespace
- Try a different web browser
- Ensure the people who are hosting the Codespace have access to Codespaces (ie. have they used up their allocation or do they have current access to Codespaces)
- are you connected to the Codespaces server (ie. are you in a region when the Codespaces are currently working; changing the server can work for this)
- as @kooya3 mentioned, if you don't own it, reach out to the maintainers to ensure they have the Codespace up and running
